demo

```diff
@@ -82,22 +111,24 @@ demo.run() {
 
 demo.help() {
   cat <<-EOF
-USAGE: 
- demo [command]
-
-COMMANDS:
- prep           creates project and service account
- setup          creates tasks, pipeline and resources
- run            starts pipeline
+		USAGE:
```

Comment:
this whitespace/tab can be ignored and will be ignored but **`-`** in `cat <<-EOF`